doc/source/roadmap-detailed.rst

```diff
@@ -355,15 +355,28 @@ functions, and spheroidal wave functions. Three possible ways to handle this:
 stats
 `````
 
-This module is in good shape overall.  New functionality that's similar to
-what's already present can continue to be added; more advanced statistical
-routines may fit better in ``statsmodels``.  Some ideas for new contributions
-are:
-
-- Implementing (well-known) distributions to the ``stats.distributions``
-  framework is always welcome.
-- Continuing work on making the function signatures of ``stats`` and
-  ``stats.mstats`` more consistent, and adding tests to ensure that that
+The ``scipy.stats`` subpackage does not seek to duplicate the advanced 
+functionality of downstream packages (e.g. StatsModels, LinearModels, PyMC3),
+but to provide a solid foundation for them. The following improvements will 
+help SciPy better serve this role.
+
+- Add fundamental and widely used hypothesis tests, including the various types
+  of ANOVA and ANCOVA.
+- Where appropriate, provide more than just a p-value as the result of a 
+  statistical test; include also confidence intervals and measures of effect 
+  size in the results.
+- Ensure that fitting a probability distribution to a data set is
+  easily accomplished with methods tailored to the distribution and 
+  and with calculations to assess the quality of the fit.
+- Implement additional widely used continuous and discrete probability 
```

Comment:
which ones? most of the ones added lately are pretty esoteric, I don't think we're missing many interesting continuous ones. mixture distributions: that may be a good idea.